geonode_mapstore_client/static/mapstore/gn-translations/data.en-US.json

```diff
@@ -270,6 +270,9 @@
             "invalidUploadMessageErrorTooltip": "The upload process failed. Please check the validity of the file and try to upload it again",
             "completeUpload": "Complete the upload",
             "selectFiles": "Select files...",
+            "addUrl": "Add URL...",
+            "duplicateUrl": "URL is already present in the list of uploads",
```

Comment:
Please update all translations with:
`This URL is already present in the list of uploads`